### ISSUE TITLE
Test the scripts

### ISSUE THREAD

**Issue by Old-Shatterhand** [kalininalab/SMTB2024#34]
The code in the scripts is not tested yet. That should be done to ensure the scripts work too. One idea is to push the main code of the scripts into the main project module and test it like regular code.

**Comment by danielkorkin** [comment]
Maybe a separate branch?

**Comment by Old-Shatterhand** [comment]
Can do, but at some point, it needs to be all together on the main branch. But for developing, why not.

**Comment by danielkorkin** [comment]
Yes, sounds good and if everything goes to plan we can merge to main
